@xen-orchestra/web/src/components/backups/configuration/BackupJobGeneralInformation.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <UiCard class="backup-jobs-information">
+    <UiTitle> {{ t('general-information') }} </UiTitle>
```

Comment:
```suggestion
    <UiTitle>{{ t('general-information') }}</UiTitle>
```
There could be display issues with extra spaces if we leave it like that.